packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -11,8 +11,8 @@ export interface FieldDefinition {
   type: string;
   label?: string;
   uuid: string;
-  placeholder?: string;
-  dateFormat?: string;
+  maxDate?: string;
+  minDate?: string;
```

Comment:
You modified the interface but did not change the actual config schema. Also why did you get rid of `placeholder`?